Bottom: bc139060d1cbedc8143011d4e858a4e11e93e78d
Top:    cc2f97c7510f937f505a22186d002cf9e9371c63
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 16:08:41 +0100

Turn off autorouter debug drawing


---

diff --git a/src/autoroute.c b/src/autoroute.c
index 5c767ea..d419031 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -92,14 +92,16 @@ RCSID ("$Id$");
 #define ROUTE_VERBOSE
 */
 
+/*
 #define ROUTE_DEBUG
-#define DEBUG_SHOW_ROUTE_BOXES
+//#define DEBUG_SHOW_ROUTE_BOXES
 #define DEBUG_SHOW_EXPANSION_BOXES
-#define DEBUG_SHOW_EDGES
-#define DEBUG_SHOW_VIA_BOXES
+//#define DEBUG_SHOW_EDGES
+//#define DEBUG_SHOW_VIA_BOXES
 #define DEBUG_SHOW_TARGETS
 #define DEBUG_SHOW_SOURCES
-#define DEBUG_SHOW_ZIGZAG
+//#define DEBUG_SHOW_ZIGZAG
+*/
 
 static direction_t
 directionIncrement(direction_t dir)
@@ -4189,7 +4191,7 @@ RouteOne (routedata_t * rd, routebox_t * from, routebox_t * to, int max_edges)
       /* we should never add edges on inactive layer groups to the heap. */
       assert (is_layer_group_active[e->rb->group]);
 #if defined(ROUTE_DEBUG) && defined(DEBUG_SHOW_EXPANSION_BOXES)
-      showedge (e);
+      //showedge (e);
 #endif
       if (e->rb->flags.is_thermal)
 	{
